hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -218,6 +218,13 @@
       reset_connections: {rst_ni: "sys"},
       base_addr: "0x40120000",
     },
+    { name: "kmac"
+      type: "kmac"
+      clock_srcs: {clk_i: "main"}
+      clock_group: "trans"
+      reset_connections: {rst_ni: "sys"}
+      base_addr: "0x41120000"
```

Comment:
@tjaychen @imphil Looks like current top earlgrey memory map does not align with the expected memory map.

Reply:
yeah i think we should probably just do another PR and re-align the whole thing. 
I know we did it for bronze..time to do it here i guess.

Reply:
Do we have an "expected memory map"? If so, where?

Reply:
https://docs.google.com/spreadsheets/d/1ptfdrG5RWiYHKtjYyDO1ubGYPPlN_4YQfzRKO1G1Mls/edit#gid=0

Reply:
Can you put that into the OT team drive? (I can't even access it.)

Reply:
Moved to "Technical specification" directory. It was opened to any @opentitan.org accounts